### PR TITLE
Fix bug with multiple pages

### PR DESCRIPTION
**Pull request by apurvis**
@matl33t @rfroetscher 

i think this fixes https://github.com/lumoslabs/broadside/issues/25 though i'm checking one more thing

### CODE REVIEW

**Comment by apurvis** [comment]
yeah this should fix it

File: lib/broadside/deploy/ecs_deploy.rb [hunk 1]
```diff
@@ -381,7 +381,7 @@ def all_results(method, key, args = {})
       results = page.send(key)
 
       while page.next_token
-        page = ecs.send(method, args.merge(next_token: page.next_token))
+        page = ecs_client.send(method, args.merge(next_token: page.next_token))
```
Comment (rfroetscher) [inline lib/broadside/deploy/ecs_deploy.rb]: `public_send`

**Comment by rfroetscher** [comment]
:zap: with public_send

**Comment by apurvis** [comment]
p.s. i added a spec for this to the refactor PR: https://github.com/lumoslabs/broadside/pull/24/commits/6ab5b5e7579cad9b264248f0ce9a5e92bc8f9193

**Comment by matl33t** [comment]
🆗

**Pull request merged**